tools/clang-tidy-plugin/test/check_clang_tidy.py

```diff
@@ -20,8 +22,10 @@
     [-check-suffix=<comma-separated-file-check-suffixes>] \
     [-check-suffixes=<comma-separated-file-check-suffixes>] \
     [-std=c++(98|11|14|17|20)[-or-later]] \
+    [-clang-tidy-binary=<path/to/clang-tidy>] \
+    [optional clang-tidy arguments]
     <source-file> <check-name> <temp-file> \
-    -- [optional clang-tidy arguments]
+    -- [optional clang arguments]
```

Comment:
I think this was correct; it's really `clang-tidy` arguments.

Reply:
https://github.com/Qrox/Cataclysm-DDA/blob/4f3e2aa0d1873c07013340f4a2347dadf85fb0b8/tools/clang-tidy-plugin/test/check_clang_tidy.py#L105-L110

https://github.com/Qrox/Cataclysm-DDA/blob/4f3e2aa0d1873c07013340f4a2347dadf85fb0b8/tools/clang-tidy-plugin/test/check_clang_tidy.py#L175-L176

Arguments after `--` are passed as clang arguments to `clang-tidy`, only the arguments before `--` are clang-tidy parameters per-se.